### ISSUE TITLE
Jagged edges on Tex objects and pyglet Window not filling entire screen

### ISSUE THREAD

**Issue by Octonions** [3b1b/manim#2158]
I'm not sure if this is really a bug but it does feel like one to me.

Whenever running manimgl as `manimgl main.py MainScene -p -f`, I get 30 fps animations, even if I specify high quality on either my flags or `custom_config.yml`. I could make it 60 fps by changing [this line](https://github.com/3b1b/manim/blob/master/manimlib/scene/scene.py#L115) to `...["fps"] = 60`.

This is the code `main.py` consists of:
```py
from manimlib import *

class MainScene(Scene):
	def construct(self):
		eq1 = Tex(
			"1 + \\frac{1}{2} + \\frac{1}{3} + \\frac{1}{4} + \\frac{1}{5} + \\cdots",
		)

		self.play(Write(eq1, 4))
		self.wait()
```

The issue now is that objects like `Circle` and `Square` look fine, but not `Tex` objects. The edges of equations are usually jagged even though the SVG object has aliased edges.

I first tested this on Windows and, to make sure it wasn't something related to how the OS renders things, I tested it on my Macbook Pro as well. Same thing happens: jagged edges for Tex objects.

I'm not sure if this is a pyglet window bug or if it's truly on how the equations are being displayed.

I wish to use the presentation mode to show animations in a live presentation by advancing with the spacebar. The only issue right now is that the equations look bad and that the pyglet window (even though specifying `-f` in the flags) does not fill my entire screen, as I can see my taskbar right below it.

![image](https://github.com/user-attachments/assets/7e5dff28-92af-4fe0-a61d-c1132410749f)

**Comment by Octonions** [comment]
https://github.com/user-attachments/assets/0cfe2618-61b5-422d-b250-8cf3eb2270b9

**Comment by Octonions** [comment]
Solved on #2160 and #2161.
